content/TrueCommand/Clusters.md

```diff
@@ -85,38 +85,45 @@ There is a single option on this page.
 ### Configure Cluster
 
 Successfully creating a cluster adds a cluster widget to the **Clusters** screen and opens options to configure the new cluster.
-These options split into three screens: **SMB Network**, **Active Directory**, and **Confirmation**.
+These options split into four screens: **VIPs**, **Accociate VIPs**, **Active Directory**, and **Confirmation**.
 
-#### SMB Network
+#### VIPs
 
-The SMB Network page has options and fields added for each SCALE system in the cluster.
+The VIPs page has options and fields added for each SCALE system in the cluster.
 
-![ConfigureClusterSMBNetwork](/images/TrueCommand/2.2/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
-
-{{< expand "Option descriptions (click to expand)" "v" >}}
+![ConfigureClusterSMBNetwork](/images/TrueCommand/2.3.1/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
 
 | Setting | Description |
 |---------|-------------|
 | **NEXT** | Click button to continue to the next page. |
```

Comment:
this appears to be on all the screens so I'd pull it out of the table and make it a new paragraph/line below the above that says something like, Next advances to the next screen.

Reply:
The NEXT button was already in one of the tables when I got started so I thought I would match it. I don't think they actually need to be there at all, so I'm going to remove them.